docs/source/terms_and_model.rst

```diff
@@ -342,11 +342,122 @@ Location for Variation.
 **Implementation Guidance**
 
 * Location refers to a position.  Although it MAY imply a sequence,
-  the two concepts are not interchangable, especially when the
+  the two concepts are not interchangeable, especially when the
   location is non-specific (e.g., a range) or symbolic (a gene).
 
 
+.. _chromosome-location:
+
+ChromosomeLocation
+$$$$$$$$$$$$$$$$$$
+
+**Biological definition**
+
+Imprecise chromosomal locations based on named landmarks assayed
+through chromosomal staining, FISH, recombination studies, or other
+low-resolution methods.
+
+**Computational definition**
+
+A ChromosomeLocation is a :ref:`Location` that is defined by a named
+chromosomal band within a species.
+
+**Information model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
+   * - _id
+     - :ref:`CURIE`
+     - 0..1
+     - Location Id; MUST be unique within document
+   * - type
+     - string
+     - 1..1
+     - Location type; MUST be set to **'ChromosomeLocation'**
+   * - species
+     - :ref:`CURIE`
+     - 1..1
+     - An external reference to a species taxonomy.  See Implementation Guidance, below.
+   * - chr
+     - string
+     - 1..1
+     - The symbolic chromosome name
+   * - start
+     - string
+     - 1..1
+     - The start chromosomal band name
+   * - end
+     - string
+     - 1..1
+     - The end chromosomal band name
```

Comment:
I know we discussed keeping `start` and `end` as separate attributes vs. implementing them as an `Interval`, but I do not remember the rationale for it (I believe it was to keep them separate for simplicity).  For the record, I prefer maintaining parallelism by packaging them as a type of `Interval`, but will not object if I am an outlier on this one.

Reply:
A cytoband is an "interval". Cytogenetic annotations can expand their fuzziness by dropping levels. For chromosome 8, this could be

* q
* q2
* q24
* q24
* q24.2
* q24.21

... by which you've arrived at good banding resolution around _MYC_ (~4Mb).

Reply:
@mbaudis Right... cytobands represent an interval, but they are not currently modeled as an `Interval` (which bundles start/end for the coordinate-based Location)

Reply:
The major reason for not making these a subclass of Interval is that it would open the possibility that people would use a sequence interval on chromosome reference, or vice versa.

A secondary reason is lack of need (IMO). Whereas SequenceLocations really will have different kinds of intervals, I don't think that the same is true here, so the complexity isn't easily justified.

Admittedly, it's not that much complexity to create a ChromosomeInterval class with start and end attributes (strings). Do you want that reconsidered?

Reply:
TODO: Based on discussion w/Alex and Larry, create NamedInterval in lieu of ChromosomeInterval

Reply:
Done. See #205.